src/io/pithos/store.clj

```diff
@@ -17,16 +17,24 @@
 (defn cassandra-store
   "Connect to a cassandra cluster, and use a specific keyspace.
    When the keyspace is not found, try creating it"
-  [{:keys [cassandra-options cluster keyspace hints repfactor username password]}]
+  [{:keys [cassandra-options cluster keyspace hints repfactor username password tls tls-options] :as config}]
   (debug "building cassandra store for: " cluster keyspace hints)
   (let [hints   (or hints
                     {:replication {:class              "SimpleStrategy"
                                    :replication_factor (or repfactor 1)}})
         cluster (if (sequential? cluster) cluster [cluster])
+        tls (or tls false)
+        tls-options (or tls-options
+                        ;; List of Mozilla's recommended suites for TLSv1.2(maximum supported in Java 8)
+                        {:cipher-suites ["TLS_ECDHE_ECDSA_WITH_AES_128_GCM_SHA256" "TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256" "TLS_ECDHE_ECDSA_WITH_AES_256_GCM_SHA384" "TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384" "TLS_DHE_RSA_WITH_AES_128_GCM_SHA256" "TLS_DHE_RSA_WITH_AES_256_GCM_SHA384"]
+                         :keystore-path (str (. (. System getProperties) (get "java.home")) "/lib/security/cacerts")
+                         :keystore-password "changeit"})
```

Comment:
password hardcode?

Reply:
Unfortunately, it is the default(!!!) password for Java truststore.

Reply:
ok, let's keep this for now as a temp solution.

Reply:
I mean it is default parameters for default truststore. Noone stops you to use your trustore and configure access to it in config.yaml file. The same for path and suites list.